### DISCUSSION TITLE
Icon for German Magenta TV

### DISCUSSION THREAD

**Discussion by GunterAv** [Mariusthvdb/custom-icons#49] [Icon suggestions/requests]
Hello,

here is my Icon for Magenta TV for Custom icon library.
![magentaTV](https://user-images.githubusercontent.com/87611027/221357946-1e085a47-1595-424c-ac49-720c77eb3abf.svg)

Best regards 
Gunter

**Comment by Mariusthvdb** [comment]
please add icon according to the default standards in size etc?
not a plain and very tiny image....
also make sure there are no copyright issues.

**Reply by GunterAv** [reply]
Hello,

??? 
The Icon is a .svg (24x24)

Best regards
Gunter

**Comment by Mariusthvdb** [comment]
for icon suggestions, check the PR's and how this is done.
